### PR TITLE
Some stuff in bank $77 (and exporting everything in home)

### PR DESCRIPTION
**Pull request by ZoomTen**

### CODE REVIEW

File: banks/bank_77.asm [hunk 1]
```diff
@@ -1,18 +1,409 @@
...
+
+asm_077_4307:
+	xor a
+	ldh [hFFBF], a
+	ld [wd0fa], a
+	jp Func_023b
+	ret
+
+Intro_Delay:
+	ld bc, $80 ; ??
```
Comment (libjet) [inline banks/bank_77.asm:396]: It's clear that they intended to loop this delay for $80 times, but the `cp 0` breaks this.
I think the question marks would be better off next to the `cp 0.`

File: banks/bank_77.asm [hunk 2]
```diff
@@ -1,18 +1,409 @@
...
+	ld [wdcfc], a
+	ld [wdcf5], a
+	ld [wdcf7], a
+	ld [wdcf8], a
+	ld [wcd40], a
+	ld [wcd41], a
+
+	ld hl, vBGMap0
+	ld de, VastFameTilemap
+	ld bc, $1412
```
Comment (libjet) [inline banks/bank_77.asm:25]: It's better to use the `lb` macro here, as the values seem to be independent as shown below.

File: banks/bank_77.asm [hunk 3]
```diff
@@ -1,18 +1,409 @@
...
+	ld bc, $40
+	call CopyBytes3
+	ld hl, VastFameGFX
+
+	ld de, vTiles2
+	ld bc, $03e0
+	call CopyBytesVRAM
+
+	ld a, $c7
+	ldh [rLCDC], a
```
Comment (libjet) [inline banks/bank_77.asm:52]: $c7 = `LCDCF_ON | LCDCF_WIN9C00 | LCDCF_OBJ16 | LCDCF_OBJON | LCDCF_BGON`

File: banks/bank_77.asm [hunk 4]
```diff
@@ -1,18 +1,409 @@
...
+	ld [wdcfb], a
+	ld [wdce8], a
+	ld [wdcf5], a
+
+.NewScreen:
+	call Func_077_4943
+	call Func_077_52a6
+	ld a, $e7
+	ldh [rLCDC], a
+	ld hl, $cab0
```
Comment (libjet) [inline banks/bank_77.asm:171]: I think this is a WRAM label

File: banks/bank_77.asm [hunk 5]
```diff
@@ -130,10 +524,110 @@ Func_077_4482:
...
+
+unk_077_7167:
+	dw $7BDE
+	dw $7E80
+	dw $6C00
+	dw 0
+	dw $7BDE
+	dw $5294
+	dw $294A
+	dw 0
```
Comment (libjet) [inline banks/bank_77.asm:627]: Hex values are typically in lowercase

**Review by pizdex — COMMENTED** [review]
Some minor changes in bank 77 needed, but otherwise it's good!

**Pull request merged**